src/Tools/dotnet-monitor/dotnet-monitor.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <RootNamespace>Microsoft.Diagnostics.Tools.Monitor</RootNamespace>
+    <ToolCommandName>dotnet-monitor</ToolCommandName>
+    <Description>.NET Core Diagnostic Monitoring Tool</Description>
+    <PackageTags>Diagnostic</PackageTags>
+    <PackageReleaseNotes>$(Description)</PackageReleaseNotes>
```

Comment:
Not setting `IsShipping` to false could end up promoting this to nuget in a full stack build.

Reply:
There's another pull request marking this as non-shipping